backends/beats/configuration.go

```diff
@@ -38,6 +38,7 @@ type BeatsConfig struct {
 	Container           interface{}       // holds the configuration object for un/marshalling
 	ContainerKeyMapping map[string]string // keys can be renamed before the configuration is rendered
 	Snippets            []beatSnippet
+	Version             []int // Beats collector version
```

Comment:
https://github.com/Graylog2/collector-sidecar/pull/98/commits/4869ab5d9aaa6679c011ddc0ebd9932aa098439f#diff-471ec5376001a4890a62eb6639226afaR139

And I wanted to report that back to the server to allow the user to detect hosts with outdated collectors